java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/buildsteps/RuntimeImageBuildStep.java

```diff
@@ -61,7 +62,8 @@ public void run(BuildContext buildContext) throws BuildStepException {
     // compat runtime requires a special app destination
     String artifactDestination = baseRuntimeImage.equals(compatImageName)
         ? "/app/"
-        : "$APP_DESTINATION";
+        : (artifact.getType() == EXPLODED_WAR ? "$APP_DESTINATION_EXPLODED_WAR"
+            : "$APP_DESTINATION");
```

Comment:
That wouldn't work for the openjdk image, since it only has APP_DESTINATION. I could add specific check just for openjdk, but I felt it was ok to do this since I also included APP_DESTINATION = APP_DESTINATION_WAR in the tomcat and jetty images.

Reply:
Ok, that's fine then. I didn't realize you had APP_DESTINATION = APP_DESTINATION_WAR.

Reply:
Changes are up, PTAL :)